pallet-fuso-verifier/src/lib.rs

```diff
@@ -478,7 +481,7 @@ pub mod pallet {
         }
 
         #[pallet::weight(T::SelfWeightInfo::verify())]
-        pub fn verify(
+        pub fn check(
```

Comment:
why?

Reply:
The benchmark macro defines verify as a keyword, this place is only to avoid conflicts when generating weights.Once the benchmark work is done, the place will be restored back.

Reply:
The benchmark macro defines verify as a keyword, this place is only to avoid conflicts when generating weights.Once the benchmark work is done, the place will be restored back.

Reply:
no, you shouldn't change the extrinsic name anyway, because the name is exposed and in-use. and the benchmark will be reserved for a long time as a part of this repo.